test/test_grid_file.py

```diff
@@ -675,6 +675,23 @@ def test_context_manager(self):
         with GridOut(self.db.fs, infile._id) as outfile:
             self.assertEqual(contents, outfile.read())
 
+    def test_exception_file_non_existence(self):
+        contents = b"Imagine this is some important data..."
+
+        try:
```

Comment:
Instead of try/except it would be cleaner to use `self.assertRaises(CorruptGridFile):` https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises